agile_bdd/spec/features/websearch_spec_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+require 'websearch'
+
+describe  "My search page" do
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.